src/main/java/com/smartbear/readyapi/client/assertions/GroovyScriptAssertionBuilder.java

```diff
@@ -0,0 +1,7 @@
+package com.smartbear.readyapi.client.assertions;
+
+import com.smartbear.readyapi.client.model.GroovyScriptAssertion;
+
+public interface GroovyScriptAssertionBuilder extends AssertionBuilder<GroovyScriptAssertion> {
```

Comment:
Otherwise I will have to change the return type in Assertions class to the implementing class to be able to expose named() method, and if I change the type to actual implementation class then it will expose unnecessary methods like build() etc.